lib/interceptor.js

```diff
@@ -8,11 +8,25 @@ var interceptor = {
     }
     done(window[NAMESPACE]);
   },
+
+  excludeUrls: function excludeUrls(urls, done) {
+    var NAMESPACE = '__webdriverajax';
+    if (window[NAMESPACE] != undefined) {
+      window[NAMESPACE].excludedUrls =
+        window[NAMESPACE].excludedUrls.concat(urls);
```

Comment:
```suggestion
      Array.prototype.push.apply(window[NAMESPACE].excludedUrls, urls);
```

Reply:
Yes. This is what I was looking for. I tried excludedUrls.push(urls), but that pushed the array as an element which is not what I wanted. Thanks.